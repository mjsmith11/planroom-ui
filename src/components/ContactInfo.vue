<template>
    <div class="row">
        <div class="col-lg-2 offset-lg-4 col-md-12">
            <div class="info"><font-awesome-icon icon="map-marker-alt" />&nbsp;&nbsp;318 E. Monroe St.</div>
            <div class="info">Kokomo, IN 46901</div>
        </div>
        <div class="col-lg-2 col-md-12">
            <div class="info"><font-awesome-icon icon="phone" />&nbsp;&nbsp;(765) 452-9500</div>
            <div class="info"><font-awesome-icon icon="fax" />&nbsp;&nbsp;(765) 452-9505</div>
        </div>
    </div>
</template>
<style scoped>
    .info {
        text-align: center;
        font-size: 14px;
        font-family: Georgia, 'Times New Roman', Times, serif;
        margin: 3px 3px;
        color: #515151;
    }
    .row {
        margin-top: 50px;
        border-top: 2px solid #fbfbfb;
        padding-top: 25px;
        margin-bottom: 20px;
    }
</style>
