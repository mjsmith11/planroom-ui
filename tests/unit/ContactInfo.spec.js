import { mount } from '@vue/test-utils'
import ContactInfo from '@/components/ContactInfo'

describe('Contact Info', () => {
    it('has correct html structure', () => {
        let cmp = mount(ContactInfo, {
            stubs: [
                'font-awesome-icon'
            ]
        })
    expect(cmp.html()).toMatchSnapshot()
    })
})