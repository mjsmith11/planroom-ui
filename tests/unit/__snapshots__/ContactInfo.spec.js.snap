// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Info has correct html structure 1`] = `
<div class="row">
  <div class="col-lg-2 offset-lg-4 col-md-12">
    <div class="info">
      <font-awesome-icon-stub icon="map-marker-alt"></font-awesome-icon-stub>&nbsp;&nbsp;318 E. Monroe St.</div>
    <div class="info">Kokomo, IN 46901</div>
  </div>
  <div class="col-lg-2 col-md-12">
    <div class="info">
      <font-awesome-icon-stub icon="phone"></font-awesome-icon-stub>&nbsp;&nbsp;(765) 452-9500</div>
    <div class="info">
      <font-awesome-icon-stub icon="fax"></font-awesome-icon-stub>&nbsp;&nbsp;(765) 452-9505</div>
  </div>
</div>
`;
